packages/common/src/sites/_internal/_ensure-event-list-card.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  IModel,
+  IDraft,
+  getProp,
+  cloneObject,
+  deleteProp,
+  setProp,
+} from "../..";
+
+/**
+ * Add telemetry config object
+ * @private
+ * @param {object} model Site Model
+ * @returns {object}
+ */
+export function _ensureEventListCard<T extends IModel | IDraft>(model: T): T {
+  const clone = cloneObject(model);
+  clone.data.values.layout.sections.forEach(
+    (section: any, sectionIndex: number) => {
+      section.rows.forEach((row: any, rowIndex: number) => {
+        row.cards.forEach((card: any, cardIndex: number) => {
```

Comment:
We only want this to run if the migration has not already been applied.

Additionally, we could simplify this a bit using `Array.prototype.map`, then we wouldn't need to leverage indexes and traversing long object paths which improves readability.

If we apply the migration, we need to bump the `schemaVersion` to `1.6`
```ts
export function _migrateEventListCardConfigs<T extends IModel | IDraft> (model: T): T {
  // do nothing if migration already applied
  if (model.item.properties.schemaVersion >= 1.6) return model;
  
  // apply migration
  const clone = cloneObject(model);
  clone.data.values.layout.sections.map(section => ({
    ...section,
    rows: section.rows.map(row => ({
      ...row,
      cards: row.cards.map(card => {
        if (card.component.name === 'event-list-card') {
          card.component.settings.displayMode = card.component.settings.calendarEnabled
            ? 'calendar'
            : 'list';
          delete card.component.settings.calendarEnabled;
        }
        return card;
      }),
    })),
  }));

  // increment schemaVersion
  clone.item.properties.schemaVersion = 1.6;

  return clone;
}
```